client/components/fields/LinkField.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { Field } from 'redux-form'
+import { connect } from 'react-redux'
+import * as selectors from '../../selectors'
+import { InputField } from './index'
+import { request } from 'axios'
+import './style.scss'
+
+export class LinkFieldComponent extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = { title: null }
+        this.errorTitle = 'Could not load title'
+        this.setTitle(this.props.link)
+    }
+
+    componentWillReceiveProps(nextProps) {
+        this.setTitle(nextProps.link)
+    }
+
+    setTitle(link) {
+        if (!link || link === 'Paste link') {
+            this.setState({title: null})
+            return
+        }
+
+        if (!this.props.iframelyKey) {
+            this.setState({title: this.errorTitle})
+            return
+        }
+
+        request({
```

Comment:
You are using `axios` without requiring it in `packages.json`.
I didn't know this library. I was using [Fetch with polyfill](https://www.npmjs.com/package/whatwg-fetch), a standard in recent browsers. Do you know it? Do you prefer `axios` ?